ocaml/src/tools/hashes.ml

```diff
@@ -86,7 +86,8 @@ class crc32c_hasher =
         self # update_substring s 0 (String.length s)
 
       method update_lwt_bytes lwt_bytes pos len =
-        cur <- Alba_crc32c.Crc32c.big_array ~crc:cur lwt_bytes pos len false
+        if len > 0
```

Comment:
Having the test here does not lead into the stubs part... (while running maintenance tests zero length arrays were passed in quite often)